internal/cmd/config/config.go

```diff
@@ -473,6 +473,15 @@ func parseEventing(eventObj *ast.ObjectItem) (*event.EventerConfig, error) {
 			s.StderrConfig = new(event.StderrSinkTypeConfig)
 		}
 
+		// parse the duration string specified in a file config into a time.Duration
+		if s.FileConfig != nil && s.FileConfig.RotateDurationHCL != "" {
+			var err error
+			s.FileConfig.RotateDuration, err = time.ParseDuration(s.FileConfig.RotateDurationHCL)
```

Comment:
We're not incredibly consistent here -- we should probably update other usages -- but can you use `parseutil.ParseDurationSecond` instead? This allows a bare value without a suffix to be treated as seconds instead of nanoseconds.